spec/requests/claims_spec.rb

```diff
@@ -25,18 +25,20 @@
   end
 
   describe "claims#create request" do
-    it "creates a new Claim for the policy and redirects to the QTS question" do
-      expect { start_student_loans_claim }.to change { Claim.count }.by(1)
+    Policies.all.each do |policy|
+      it "creates a new Claim for the policy and redirects to the next question in the sequence" do
```

Comment:
It'd be good to have the policy referenced in the name of the spec here

Reply:
Good shout.